model/code/registry.yaml

```diff
@@ -7,38 +7,38 @@ groups:
     attributes:
       - id: code.function
         type: string
-        stability: experimental
+        stability: experimental # RELEASE CANDIDATE
```

Comment:
I don't know why, but yamllint wants two spaces before the `#` (you can see these warnings if you look at the "Files changed" tab on the PR)